Block/Form.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Bolt\Boltpay\Block;
+
+use Bolt\Boltpay\Helper\Config;
+use Magento\Backend\Model\Session\Quote as BackendQuote;
+use Magento\Payment\Block\Form as PaymentForm;
+use Magento\Framework\View\Element\Template\Context;
+
+/**
+ * Class Form
+ */
+class Form extends PaymentForm
```

Comment:
Is this used by both admin and non-admin? If so, please add comment. If it's admin only, maybe rename to AdminForm?

Reply:
I think we can refactor later; not blocking.

Reply:
As i see this block use for both part admin and frontend.
The calling of this block is defined in payment model and this model calls in both part.

>app/code/Bolt/Boltpay/Model/Payment.php

```php

    /**
     * @var string
     */
    protected $_formBlockType = \Bolt\Boltpay\Block\Form::class;

```